config/application.rb

```diff
@@ -29,7 +29,7 @@ class Application < Rails::Application
     # These settings can be overridden in specific environments using the files
     # in config/environments, which are processed later.
     #
-    # config.time_zone = "Central Time (US & Canada)"
+    # config.time_zone = "Europe/Warsaw"
```

Comment:
you changed timezone but it still commented

Reply:
I had some strange things happening when I uncommented it so I commented it again :D I have to come back to that.